Source/WebKit/UIProcess/API/glib/WebKitSecurityOrigin.cpp

```diff
@@ -223,8 +233,10 @@ guint16 webkit_security_origin_get_port(WebKitSecurityOrigin* origin)
  * webkit_security_origin_is_opaque:
  * @origin: a #WebKitSecurityOrigin
  *
+ * This function returns %FALSE.  
+ *
  * This function returns %FALSE. #WebKitSecurityOrigin is now a simple
- * wrapper around a &lt;protocol, host, port&gt; triplet, and no longer
+ * wrapper around a <protocol, host, port> triplet, and no longer
```

Comment:
Good catch as well, no need to use SGML entities (the `&stuff;` things) with `gi-docgen`. This makes the documentation easier to read and edit when making changes to the code.